Mage/src/main/java/mage/abilities/dynamicvalue/common/CardsInControllerGraveyardCount.java

```diff
@@ -27,6 +28,7 @@ public CardsInControllerGraveyardCount(FilterCard filter) {
     }
 
     public CardsInControllerGraveyardCount(FilterCard filter, Integer multiplier) {
+        // TODO: this class doesnt need a multiplier. use MultipliedValue instead
```

Comment:
26 usages... same trick with "null"

Reply:
Linking related conversation for my reference https://github.com/magefree/mage/pull/12596#discussion_r1739939202